qiskit_dynamics/backend/dynamics_backend.py

```diff
@@ -351,6 +374,159 @@ def target(self) -> Target:
     def meas_map(self) -> List[List[int]]:
         return self.options.meas_map
 
+    def configuration(self) -> PulseBackendConfiguration:
+        """Get the backend configuration."""
+        return self.options.configuration
```

Comment:
Should these be calculated from target, otherwise you could have different value depending on if you access info via target or configuration/defaults